### PR TITLE
Cockpit 155

### PR DESCRIPTION
**Pull request by martinpitt**
Release notes for 155.

Preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-155.html

### CODE REVIEW

**Comment by martinpitt** [comment]
Force-pushed to add a paragraph about the OpenShift 3.7 change that just landed. Release is running, but not done yet.

**Comment by martinpitt** [comment]
Release 155 is done now, so good to review/land.

File: _posts/2017-11-06-cockpit-155.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 155.
+
+### NFS client support
+
+The Storage page now shows currently mounted NFS volumes and offers to
+mount/unmount them.
```
Comment (stefwalter) [inline _posts/2017-11-06-cockpit-155.md]: Worth saying that authentication/kerberos is not supported yet?

Comment (martinpitt) [reply _posts/2017-11-06-cockpit-155.md]: Yes, that can't hurt. Pushed.

**Pull request merged**